app/src/main/java/com/exuberant/code2create/adapters/AgendaAdapter.java

```diff
@@ -9,14 +10,24 @@
 import com.exuberant.code2create.R;
 import com.exuberant.code2create.models.Agenda;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalTime;
+import java.util.Calendar;
+import java.util.Date;
 import java.util.List;
+import java.util.Locale;
 
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
 
 public class AgendaAdapter extends RecyclerView.Adapter<AgendaAdapter.AgendaAdapterViewHolder> {
 
     List<Agenda> agendaList;
+    String endTime;
+    String startTime;
+    Integer var2,var1;
```

Comment:
what is var1 and var2? Name variables appropriately

Reply:
Removed